lachesis-rs/src/server/http_handler.rs

```diff
@@ -75,12 +75,12 @@ pub fn check_transaction_status(
     req: &HttpRequest<AppState>,
 ) -> Box<Future<Item = HttpResponse, Error = Error>> {
     //TODO: implement get transaction status from id
```

Comment:
Can we create issues out those todo?

Reply:
yep. these todos link to issue #51 to some degree but this one needs a some sort of transaction handling code to be written that links with the the lachesis code.